docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb

```diff
@@ -185,6 +185,158 @@
     "    ![stac_ipyleaflet aoi bbox](../../_static/stac_ipyleaflet_bbox.png)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    #!pip install ipyleaflet
</pre><p>This isn't necessary with stac_ipyleaflet, because it's already built in.</p><p> </p><p>And I believe the other imports (httpx and shapely) should already be available too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Are you suggesting I delete the 
```
"#!pip install httpx\n",
"#!pip install ipyleaflet\n",
"#!pip install shapely"
``` 
line? I added it because I needed to run these commands in order to get the notebook to run for me in ops ADE
The `from ipyleaflet import TileLayer` line comes before installing stac_ipyleaflet, so that is why I needed it 
@emmalu Curious if you didn't need these installs

Reply:
@grallewellyn were you using the Pangeo workspace? stac_ipyleaflet hasn't been pushed to the other images yet. Also it's not reliable to pip install it.

Reply:
I was using the Pangeo workspace. Should I add that to this notebook? 
I didn't need to pip install stac_ipyleaflet

Reply:
I am able to start a new workspace using the Pangeo image, create a new notebook and confirm that all 3 of the libraries (httpx, shapely & ipyleaflet) are preinstalled - so I think the only `pip install` that is needed is for using the `cogeo_mosaic` library - which is documented in [the other notebook](https://app.reviewnb.com/MAAP-Project/maap-documentation/blob/feature/remove-ipycmc/docs/source/technical_tutorials/visualization/visualize_mosaic_stac_ipyleaflet.ipynb/).
![Screenshot 2023-09-21 at 5 22 05 PM](https://github.com/MAAP-Project/maap-documentation/assets/18629881/c5ed1058-a8f9-4df0-97b9-057cf02796d1)

Reply:
Deleted
"#!pip install httpx\n",
"#!pip install ipyleaflet\n",
"#!pip install shapely"
 and pushed changes, thank you for the suggestion!

Reply:
Great! Happy to help simplify. 👍🏽